docs/.vitepress/config.js

```diff
@@ -607,22 +603,19 @@ function sidebar() {
 			collapsible: true,
 			collapsed: true,
 			items: [
-				{
-					link: '/contributing/introduction',
-					text: 'Introduction',
-				},
-				{
-					link: '/contributing/running-locally',
-					text: 'Running Locally',
-				},
-				{
-					link: '/contributing/codebase-overview',
-					text: 'Codebase Overview',
-				},
-				{
-					link: '/contributing/tests',
-					text: 'Tests',
-				},
+				{ link: '/contributing/introduction', text: 'Introduction' },
+				{ 
+					text: 'Code',
+					items: [
+						{ link: '/contributing/feature-request-process', text: 'Request a Feature' },
```

Comment:
I feel like feature requests should fall outside of the "code" scope and more in the community scope since non-developers / non-code users can make feature requests and it is open to the community. Thoughts?